challenge_4/python/CindyB/src/TreeNode.py

```diff
@@ -0,0 +1,37 @@
+class TreeNode:
+
+    def __init__(self,value):
+        self.value = value
+        self.rightChild = None
+        self.leftChild = None
+
+    def GetValue(self):
```

Comment:
Due to how Python and it's properties work, there's no need to create getters / setters to start as they can be created later if necessary without modifying the callers. Instead the class variables should be directly used for such a simple use case.